packages/material-ui/src/Modal/ModalManager.js

```diff
@@ -83,7 +87,7 @@ function handleNewContainer(containerInfo) {
 
     [].forEach.call(fixedNodes, node => {
       const paddingRight = getPaddingRight(node);
-      restorePaddings.push(paddingRight);
+      restorePaddings.push(getPaddingRightRaw(node));
```

Comment:
The restore logic doesn't seem to be consistent between the .mui-fixed class names and the body element. What's the best approach (.style.x vs getComputedStyle)?

Reply:
computed style triggers a reflow and would possible add styling. 

Can we restore the full style object instead of cherry-picking values?

Reply:
> computed style triggers a reflow and would possible add styling.

So .style.x? I can't remember why I have used `css` from dom-helpers in the first place.

Reply:
> So .style.x?

yes

Reply:
Are we requesting that we store the entire style object and just go `node.style = previousStyleObject`?

Reply:
> Are we suggesting that we store the entire style object and just go node.style = previousStyleObject?

That was my intuition at least. Though I don't have the context for the decision to cherry-pick restoration.

Reply:
Would it work, is the CSSStyleDeclaration object immutable?